### PR TITLE
Html

### PR DESCRIPTION
**Pull request by karl-police**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,12 +10,12 @@
 			<h1 id="header_title" onClick="window.location.href='index.html';">Steam Tools</h1>
 		</header>
 		<div id="button_menu">
-			<a class="btn_menu button_menu1" href="html/friends_offlinesorter.html" style="background: blue;">Offline Friends Sorter</a>
+			<a class="btn_menu button_menu1" href="steamtools/friends_offlinesorter.html" style="background: blue;">Offline Friends Sorter</a>
```
Comment (verlegend454-a11y) [inline index.html:13]: Tugxtj
R6rw3
Dt[uufg
Dggdug9
Glgud6fl
Figures
Uyrtj
VERlegend454

File: index.html [hunk 2]
```diff
@@ -10,12 +10,12 @@
 			<h1 id="header_title" onClick="window.location.href='index.html';">Steam Tools</h1>
 		</header>
 		<div id="button_menu">
-			<a class="btn_menu button_menu1" href="html/friends_offlinesorter.html" style="background: blue;">Offline Friends Sorter</a>
+			<a class="btn_menu button_menu1" href="steamtools/friends_offlinesorter.html" style="background: blue;">Offline Friends Sorter</a>
 			<a class="btn_menu button_menu2" href="#">Test</a>
```
Comment (verlegend454-a11y) [inline index.html:14]: class="btn_menu
button_menu2"
href="#">Test</a>

**Review by verlegend454-a11y — COMMENTED** [review]
This actually works

File: index.html (hunk 2 above)
Comment (bambari17-alt) [reply index.html:14]: No

Comment (bambari17-alt) [reply index.html:14]: Idk u ow..

**Pull request merged**